pwned-check/README.md

```diff
@@ -0,0 +1,74 @@
+# Pwned credentials check
+
+Module that provides functionality of checking if the credentials being used for login attempt have been compromised in a data breach.
+
```

Comment:
I would also add a short description of how this check is performed. On a local database, from bloom file, from hashes obtained from any credentials data breach. I would specify that it can be any file that can be generated.